docs/documentation/authorization_services/topics/enforcer-js-adapter.adoc

```diff
@@ -19,10 +19,9 @@ import Keycloak from "keycloak-js";
 import KeycloakAuthorization from "keycloak-js/authz";
 
 const keycloak = new Keycloak({
-    // Replace this with your own configuration.
```

Comment:
wouldn't it be nice to keep this line although it's pretty obvious

Reply:
Indeed, I removed it for consistency with the other places the configuration is shown. I think a user should be able to figure it out by reading the code, or at the very least by running it and getting an error.